.zenodo.json

```diff
@@ -6,7 +6,7 @@
     "upload_type": "software",
     "creators": [
         {
-            "affiliation": "CERN",
+            "affiliation": "Technical University of Munich",
```

Comment:
@lukasheinrich Technical University of Munich or Technische Universität München?